fulp_modules/bloodsuckers/code/antagonists/bloodsucker_datum.dm

```diff
@@ -266,7 +266,11 @@
 			H.dna.remove_mutation(CLOWNMUT)
 			to_chat(H, "As a vampiric clown, you are no longer a danger to yourself. Your clownish nature has been subdued by your thirst for blood.")
 	// Physiology
-	CheckVampOrgans() // Heart, Eyes
+	CheckVampOrgans() // Heart, Tongue
+	// Makes eyes flash-immune
+	var/obj/item/organ/eyes/E = owner.current.getorganslot(ORGAN_SLOT_EYES)
+	E.flash_protect = -1
```

Comment:
Bloodsuckers weren't flashproof before right?

Reply:
It's -1 flash protection, because they get moth eye-type protection. Masquerade temporarily hides this